internal/pkg/es/client.go

```diff
@@ -6,21 +6,57 @@ package es
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"net/http"
 	"runtime"
+	"syscall"
+	"time"
 
 	"go.elastic.co/apm/module/apmelasticsearch/v2"
 
 	"github.com/elastic/fleet-server/v7/internal/pkg/build"
 	"github.com/elastic/fleet-server/v7/internal/pkg/config"
 	"github.com/rs/zerolog"
 
+	backoff "github.com/cenkalti/backoff/v4"
 	"github.com/elastic/go-elasticsearch/v8"
 )
 
+const (
+	initialRetryBackoff = 500 * time.Millisecond
+	maxRetryBackoff     = 10 * time.Second
+	randomizationFactor = 0.5
+	defaultMaxRetries   = 5
+)
```

Comment:
Is there some official guidance from ES where we're getting these values from?

Reply:
not official i went over repos talking to ES manually